src/update_handler.c

```diff
@@ -684,6 +684,8 @@ static gboolean run_slot_hook(const gchar *hook_name, const gchar *hook_cmd, Rau
 
 		g_subprocess_launcher_setenv(launcher, "RAUC_BUNDLE_SPKI_HASHES", string, FALSE);
 		g_free(string);
+
+		g_subprocess_launcher_setenv(launcher, "RAUC_UPDATE_SOURCE", bundle->mount_point, TRUE);
```

Comment:
As we have `RAUC_SLOT_MOUNT_POINT` already, what about using `RAUC_BUNDLE_MOUNT_POINT` here instead?
Otherwise, I endorse the approach.

Reply:
Well, I agree that RAUC_UPDATE_SOURCE isn't the best name, and RAUC_BUNDLE_MOUNT_POINT is more obvious. But RAUC_UPDATE_SOURCE isn't my invention, that's the name used for handlers, so I thought it was best to expose the same piece of data under the same name. But if RAUC_BUNDLE_MOUNT_POINT is preferred, I guess we could use that here, and then also expose it under that name to handlers (updating the documentation, but keeping defining RAUC_UPDATE_SOURCE as well for compatibility with existing setups). Either way works for me - but ending up using different names in the handler vs. hook cases is a bad idea, IMO.

Reply:
+1 to using/adding `RAUC_BUNDLE_MOUNT_POINT` while keeping `RAUC_UPDATE_SOURCE` for backwards compatibility and explicitely stating that the latter is regarded as being obsolete.

Reply:
Yes, `RAUC_BUNDLE_MOUNT_POINT` and `RAUC_SLOT_MOUNT_POINT` is much better, let's just deprecate `RAUC_UPDATE_SOURCE`.

Reply:
Updated and force-pushed a new version with a first patch replacing/deprecating RAUC_UPDATE_SOURCE.